### PR TITLE
Allow passing other requests.get kwargs

### PR DESCRIPTION
**Pull request by Nelson-Gon**
Hi,

Thanks for the work done. This PR aims to allow usage of other `requests.get` arguments in `_request` and `_cache_or_request`. This is necessary because sometimes one may need to use the `get` method directly, for example to "bypass", at least temporarily, issues with SSL certificate verification. 
See for example this [thread](https://stackoverflow.com/questions/10667960/python-requests-throwing-sslerror)

Thank you,
NG

### CODE REVIEW

**Review by harryhaller001 — CHANGES_REQUESTED** [review]
Hi, thanks for the changes you submitted. Can your fix the linting errors in the file? For details run `pylint` in your local repo or look at the logs from the CI workflow.

**Review by Nelson-Gon — COMMENTED** [review]
I have accepted the changes as I'm having issues with `pylint` locally.

File: keggtools/resolver.py [hunk 1]
```diff
@@ -54,18 +56,19 @@ def get_gene_names(genes: List[str], max_genes: int = 50) -> Dict[str, str]:
     #         )
 
     # Build query string
-    query_string: str = "+".join(genes)
```
Comment (harryhaller001) [inline keggtools/resolver.py:57]: Hi. The plus is required here. It indicates the concatination of multiple gene identifier.

**Review by harryhaller001 — CHANGES_REQUESTED** [review]
Hi,
thanks for your submittion. The linting is correct now, but you removed the plus from the query string, so the testing for HTTP requests are failing.

File: keggtools/resolver.py [hunk 2]
```diff
@@ -54,18 +56,19 @@ def get_gene_names(genes: List[str], max_genes: int = 50) -> Dict[str, str]:
     #         )
 
     # Build query string
-    query_string: str = "+".join(genes)
+    query_string: str = " ".join(genes)
```
Comment (harryhaller001) [inline keggtools/resolver.py]: Hi. The plus is required here. It indicates the concatination of multiple gene identifier.

**Review by harryhaller001 — CHANGES_REQUESTED** [review]

File: keggtools/resolver.py (hunk 2 above)
Comment (Nelson-Gon) [reply keggtools/resolver.py]: Hi, this was removed by an autolinter I used and I missed it. Sorry, I have restored it.

**Review by harryhaller001 — APPROVED** [review]
Thanks for your final changes!

**Pull request merged**